mk/config.mk

```diff
@@ -32,8 +32,8 @@ GIT_BRANCH ?= $(shell git rev-parse --symbolic-full-name --abbrev-ref HEAD)
 
 
 # possible values : mf-geoadmin3 or mvt (in the end, anything else than mf-geoadmin3 could do the trick to switch to testviewer deploy)
-# auto-detect MVT branches if branch name contains "mvt_", otherwise falls back to standard mf-geoadmin3
-ifneq (,$(findstring mvt_,$(GIT_BRANCH)))
+# auto-detect mvt_clean branches, otherwise falls back to standard mf-geoadmin3 (same behavior as in the Jenkinsfile)
```

Comment:
Hmm...
Do we want to deploy branches for project `mvt`. If so, this is not enough
`Jenkins` deploys two branches `master`and `mvt_clean`, and `PR`. with PR, we know on which on which master it will be merged. `Makefile`has only a branch name to decide to which target to deploy.